concourse/pipeline.yml

```diff
@@ -214,6 +215,7 @@ jobs:
           CF_STARTUP_TIMEOUT: 15 # minutes
           APP_INSTANCES: 20
           WORKER_INSTANCES: 50
```

Comment:
WORKER_INSTANCES can go back down to 20 now, we only bumped it up to try to get through the password jobs

Reply:
Done here https://github.com/alphagov/govuk-account-manager-prototype/pull/566/commits/efaea0324a0252a22ea9fceb8c8e8f709963dd47